packages/react-conformance/src/defaultTests.tsx

```diff
@@ -11,6 +12,8 @@ import * as _ from 'lodash';
 import * as path from 'path';
 import consoleUtil from './utils/consoleUtil';
 
+const CALLBACK_REGEX = /^on(?!Render[A-Z])[A-Z]/;
```

Comment:
nit: This might be clearer

```suggestion
const RENDER_CALLBACK_REGEX = /^on(?!Render[A-Z])[A-Z]/;
```

Reply:
Ah nvm, it's to explicitly ignore `onRender` callbacks